examples/cloud-tomee-azure/README_pt.adoc

```diff
@@ -0,0 +1,111 @@
+- https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest[Instalar] o Azure (CLI) de acordo com o sistema operacional do computador que você está usando para desenvolver.
+
+- Finalmente, você pode configurar seu computador de desenvolvimento.
+
+=== Fazendo login no Azure
+
+`az login`
+
+O resultado:
+
+----
+[
+   {
+     "cloudName": "AzureCloud",
+     "id": "aaaaaaaa-aaaa-aaaa-aaaaa-aaaaaaaaaa",
+     "isDefault": true,
+     "name": "Pay-As-You-Go",
+     "state": "Enabled",
+     "tenantId": "bbbbbbb-bbbbb-bbbb-bbbbb-bbbbbbbbbbb",
+     "user": {
+       "name": "<o email da sua conta azure>",
+       "type": "user"
+     }
+   }
+ ]
+----
+O TenantId é alguém que pode registrar e gerenciar aplicativos por conta própria. Você precisará disso para mais tarde.
+
+
+=== Criando um service principal
+Um Azure service principal é uma identidade de segurança usada por aplicativos, serviços e
+e ferramentas de automação para acessar recursos específicos do Azure:
+
+----
+az ad sp create-for-rbac --name  http://<seu-sub-domínio> --password <password para este aplicativo>
+
+{
+  "appId": "cccccccc-cccc-cccc-cccc-ccccccccccccccc",
+  "displayName": "cloud-tomee-azure",
+  "name": "http://cloud-tomee-azure",
+  "password": "<password for this app>",
+  "tenant": "bbbbbbb-bbbbb-bbbb-bbbbb-bbbbbbbbbbb"
+}
+----
+O <seu-sub-domínio>, também chamado de service principal na documentação do Azure.
+Neste exemplo "http://cloud-tomee-azure". Tem que ser exclusivo no Azure.
+O appId é a identificação do serviço de aplicativo.
+
+=== Configurando o  Maven
+
+Você pode continuar usando apenas a CLI do Azure para autenticação, mas também podemos fazê-lo permanentemente com o maven.
+
+Nesse caso, precisamos editar o arquivo settings.xml do Maven para que o plugin azure-webapp-maven possa autenticar no Azure:
+
+Você pode adicionar um novo servidor em `~ / .m2 / settings.xml` assim:
+
+----
+<server>
+  <id>azure-auth</id>
+  <configuration>
+     <client>cccccccc-cccc-cccc-cccc-ccccccccccccccc</client>
+     <tenant>bbbbbbb-bbbbb-bbbb-bbbbb-bbbbbbbbbbb</tenant>
+     <key><password para este aplicativo></key>
+     <environment>AZURE</environment>
+   </configuration>
+</server>
+----
+É isso aí. Agora você pode criar o exemplo e implantá-lo no Azure usando o Maven:
+
+`mvn clean install -Pazure-single-jar azure-webapp:deploy`
+
+O azure-webapp é explicitamente chamado porque depende de sua conta do Azure. O build padrão do TomEE não usará uma conta do Azure.
+
+O URL final será semelhante a:
+
+`https://<your-sub-domain>.azurewebsites.net/cloud-tomee-azure-8.0.0-SNAPSHOT/echo/send-this-back`
+
+==== Notas
+
+No momento da criação deste exemplo, há um erro no azure com o JAVA_HOME que impede a implementação.
+Verificar: https://github.com/Azure-App-Service/java/issues/11
+A solução é definir o Env. variável no console da Web do Azure e reinicie o aplicativo.
```

Comment:
You can translate this "the Env. variable" to "a variável de ambiente" and change "e reinicie o aplicativo." to "e reiniciar o aplicativo."

Reply:
done. thanks.